docs/installation.md

```diff
@@ -0,0 +1,77 @@
+## Prerequisites
+
+Ensure you have Python version 3.9 or above installed on your system.
+
+We recommend creating and activating a Python virtual environment using [`venv`](https://docs.python.org/3/library/venv.html) before installing `nemos`. This practice helps manage dependencies and avoid conflicts with other Python packages.
+
+For `venv`, create your virtual environment in a specific directory. This example uses `~/python_venvs/nemos` for Linux/Mac and `C:\Users\username\python_venvs\nemos` for Windows; in general, free to use whatever directory works best for you.
+
+### Creating and Activating a Virtual Environment
+
+**For macOS and Linux:**
+
+1. Open a terminal.
+
+2. Run the following commands to create and activate the virtual environment:
+
+    ```bash
+    python -m venv ~/python_venvs/nemos
+    source ~/python_venvs/nemos/bin/activate
+    ```
+
+**For Windows:**
+
+1. Open a command prompt.
+
+2. Execute the commands below to create and activate the virtual environment:
+    ```bash
+    python -m venv C:\Users\username\python_venvs\nemos
+    C:\Users\username\python_venvs\nemos\Scripts\activate
+    ```
+    
+**Note: Replace `username` with your actual Windows username.*
```

Comment:
see comment above -- if we get the HOMEPATH variable working, I think that's the way to do it